.gitignore

```diff
@@ -1,5 +1,5 @@
-#Ignore Mac system files
-.DS_store
+.DS_Store
 
 #Ignore models in the folder
-https://drive.google.com/drive/folders/1jmSVF-PuWZd1W1kHmZ0mhea1H2iZADix?usp=sharing
\ No newline at end of file
+models/
+data/raw/
```

Comment:
It's always a good idea to have a newline at the end of all your files.

Reply:
I have pushed .py files. 
Ok. I will take care of this from next time.